rules.neon

```diff
@@ -17,3 +17,10 @@ services:
 			deprecationRulesInstalled: %deprecationRulesInstalled%
 		tags:
 			- phpstan.rules.rule
+
+	-
+		class: PHPStan\Rules\PHPUnit\AssertEqualsIsDiscouragedRule
+		arguments:
+			strictRulesInstalled: %strictRulesInstalled%
```

Comment:
I'd rather use conditionalTags instead of passing the parameter in the rule.

Also: we can do it only on bleeding edge. Conditional tags can do `and` condition based on multiple parameters since https://github.com/phpstan/phpstan-src/pull/1697 (see the tests).

Reply:
TIL, thanks